external/runtime/runtime.depproj

```diff
@@ -48,22 +48,6 @@
     </PackageReference>
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="@(RefAndRuntimePackageReference)" />
```

Comment:
question: Do you even use this RefAndRuntimePackageReference any longer? If not, you can probably clean it up now. from external/dir.props